Products/CMFPlone/controlpanel/browser/redirects.py

```diff
@@ -190,20 +191,35 @@ def __init__(self, query="", created="", manual=""):
             else:
                 manual = ""
         if created:
+            end = created
+            warnings.warn(
+                "The 'datetime' parameter is deprecated. Use 'end' parameter instead.",
```

Comment:
```suggestion
                "The 'created' parameter is deprecated. Use 'end' parameter instead.",
```

Reply:
@davisagli I was considering `datetime` as parameter `created` is being sent by `plone.restapi` as `datatime`, did you specify this because the scope of RedirectionSet is being used internally by `CMFPlone` only? And we mention the param local to the repo scope where it is being delivered?
Only asking for learning purpose, did change it do as mentioned :)

Reply:
This function accepts a parameter called `created`, so it should refer to it by that name. It doesn't know where the parameter came from. If plone.restapi accepts a different name, it should take responsibility for validating its input with a different name before passing it here.